libraries/ms-common/test/test_pwm.c

```diff
@@ -3,23 +3,26 @@
 #include "test_helpers.h"
 #include "unity.h"
 
+// TODO(ELEC-263): Consider adding functional or more rigorous testing.
```

Comment:
I think we should at least verify basic functionality - that the IO changes as you'd expect and it doesn't when not explicitly configured.